Hera Example.ipynb

```diff
@@ -77,10 +77,15 @@
    },
    "outputs": [],
    "source": [
-    "hera.global_config.GlobalConfig.token = \"<your-token>\"\n",
-    "hera.global_config.GlobalConfig.host = \"https://argo-workflows.aaw-dev.cloud.statcan.ca:443\"\n",
```

Comment:
moved this to its own constant for cleanliness and as a reminder that we should remove -dev since users should never use our dev env.